src/w3c/templates/headers.js

```diff
@@ -3,11 +3,25 @@ import showLogo from "./show-logo";
 import showPeople from "./show-people";
 import showLink from "./show-link";
 
+function decorateSpecTitle(specTitleElem) {
+  specTitleElem.classList.add("title", "p-name");
+  specTitleElem.id = "title";
+}
+
 export default conf => {
   const html = hyperHTML;
+  const specTitleElem = document.querySelector("h1#title, body>h1") || document.createElement("h1");
+  if (specTitleElem.parentElement) {
+    // remove it from the DOM tree, because we are going to relocate it
+    specTitleElem.remove();
+    conf.specTitle = specTitleElem.innerText.trim();
+  } else {
```

Comment:
We need one more check here:
```js
if(document.title !== conf.title) {
   pub("warn", "The document's title and the `<title>` element differ.") 
}
```

Reply:
Done! What all should I check in testing. That if `h1` exists what all should be in place and vice versa ?

Reply:
Also how do I perform a test on the `title` ? Aren't the tests in `tests/spec/w3c/headers-spec.js` mean't for the specific code block of 
```html
<section id='sotd'><p>foo</p></section><section id='toc'></section>
```
Shall I make a different one ?

Reply:
> Done! What all should I check in testing. That if h1 exists what all should be in place and vice versa ?

 Yes, exactly. Also, check that `conf.title` is correctly set.

Reply:
> Also how do I perform a test on the `title` ? 

Unfortunately, we won't be able to check that one :( We have a bug for that #1084.